packages/react-native-bpk-component-chip/src/BpkChipWrapper.js

```diff
@@ -85,6 +85,32 @@ const styles = StyleSheet.create({
   iconDisabled: {
     color: colorGray300,
   },
+  innerOutline: {
+    backgroundColor: 'transparent',
+    borderColor: colorGray300,
+    borderWidth: borderSizeSm,
+    paddingVertical: spacingMd - borderSizeSm,
+  },
+  innerOutlineSelected: {
+    backgroundColor: colorBlue500,
+    borderColor: colorGray300,
```

Comment:
Is this needed given that the `innerOutline` style already has it?

Reply:
Good point!